config.md

```diff
@@ -387,31 +387,102 @@ For POSIX platforms, the configuration structure supports `hooks` for configurin
         * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2008's `environ`][ieee-1003.1-2008-xbd-c8.1].
         * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
             If set, `timeout` MUST be greater than zero.
-    * **`poststart`** (array of objects, OPTIONAL) is an array of [post-start hooks](#poststart).
-        Entries in the array have the same schema as pre-start entries.
-    * **`poststop`** (array of objects, OPTIONAL) is an array of [post-stop hooks](#poststop).
-        Entries in the array have the same schema as pre-start entries.
+    * **`create-runtime`** (array of objects, OPTIONAL) is an array of [`create-runtime` hooks](#create-runtime-hooks).
+        Entries in the array contain the following properties (the entries are identical to the entries in the deprecated `prestart` hooks):
+        * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2008 `execv`'s *path*][ieee-1003.1-2008-functions-exec].
+            This specification extends the IEEE standard in that **`path`** MUST be absolute.
+            Runtimes MUST resolve this value in the [runtime namespace](glossary.md#runtime-namespace).
+        * **`args`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2008 `execv`'s *argv*][ieee-1003.1-2008-functions-exec].
+        * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2008's `environ`][ieee-1003.1-2008-xbd-c8.1].
+        * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
+            If set, `timeout` MUST be greater than zero.
+    * **`create-container`** (array of objects, OPTIONAL) is an array of [`create-container` hooks](#create-container-hooks).
+        Entries in the array have the same schema as `create-runtime` entries.
+        Note that even if the hook will be executed in the container namespace, the `path` should resolve in the runtime namespace.
+    * **`start-container`** (array of objects, OPTIONAL) is an array of [`start-container` hooks](#start-container-hooks).
+        Entries in the array contain the following properties:
+        * **`pathContainer`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2008 `execv`'s *path*][ieee-1003.1-2008-functions-exec].
```

Comment:
is there any advantage in calling it `pathContainer` instead of `path`?  It would avoid a new type as everything else seems the same as with the other hooks

Reply:
It makes it clearer in the API where the path is expected to be resolved. Since this is the only hook that is expected to be resolved in the container, I thought it made sense to make an explicit separation.

Reply:
I'd still prefer to simplify it and reuse the existing types where possible.  IMO they have the same semantic, the only difference is the mount namespace used

Reply:
Addressed in the latest update